go-controller/pkg/ovn/gateway.go

```diff
@@ -388,7 +388,10 @@ func (gw *GatewayManager) GatewayInit(
 		}
 		if gw.netInfo.IsPrimaryNetwork() && gw.netInfo.TopologyType() == types.Layer2Topology {
 			logicalRouterPort.Ipv6RaConfigs = map[string]string{
-				"address_mode": "dhcpv6_stateful",
+				"address_mode":  "dhcpv6_stateful",
+				"send_periodic": "true",
+				"max_interval":  "900", // 15 minutes
+				"min_interval":  "300", // 5 minutes
```

Comment:
what's the point of setting both options here? min and max? it seems 5mins is going to win here correct?

Reply:
Don't know, this is OVN implementation, maybe they use a random number between the two, is very common to do some kind of jitter on repeated messages, so systems do get hammered.

Reply:
@qinqon what I meant was do we need to set both just because both are available? it seems either or right?

Reply:
There's some random jitter ovn-controller adds:
```
static long long int
ipv6_ra_calc_next_announce(time_t min_interval, time_t max_interval)
{
    long long int min_interval_ms = min_interval * 1000LL;
    long long int max_interval_ms = max_interval * 1000LL;

    return time_msec() + min_interval_ms +
        random_range(max_interval_ms - min_interval_ms);
}
```
thanks @qinqon saw the response :)